src/styles/index.scss

```diff
@@ -1,3 +1,80 @@
+$calendar-columns: 7;
+$day-bdc: #eee;
+$day-hover: 'Arial, Helvetica, sans-serif';
+$cell-size: 100px;
+$gap-size: 1px;
+$paddind-calendar: 10px;
+$days-count: 10px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$calendar-width: (($cell-size * 7) + ($gap-size * 6) + (2 * $paddind-calendar));
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap-size;
+  padding: $paddind-calendar;
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+    cursor: pointer;
+    transition: 500ms;
+    &:hover {
```

Comment:
- Specify to which properties should transition apply
- Add a blank line before the selector for better readability